pkg/providers/tinkerbell/config/template-cp.yaml

```diff
@@ -49,11 +49,67 @@ spec:
         - DirAvailable--etc-kubernetes-manifests
         kubeletExtraArgs:
           provider-id: PROVIDER_ID
-    preKubeadmCommands:
-    - mkdir -p /etc/kubernetes/manifests && ctr images pull {{.kubeVipImage}}
-      && ctr run --rm --net-host {{.kubeVipImage}} vip /kube-vip manifest
-      pod --arp --interface $(ip -4 -j route list default | jq -r .[0].dev) --address {{.controlPlaneEndpointIp}} --controlplane
-      --leaderElection > /etc/kubernetes/manifests/kube-vip.yaml
```

Comment:
Was it decided to remove this for now? Or do we want to figure the root cause out first?